test/runtests.jl

```diff
@@ -47,3 +47,54 @@ end
   JtF = jtprod_residual(nls_model, x, F)
   @test all(g .≈ JtF)
 end
+
+@testset "SVM-Train" begin
+  nlp_train,nls_train,sol = svm_train_model()
+  @test typeof(nlp_train) <: ADNLPModel
+  @test typeof(nls_train) <: ADNLSModel
+  @test typeof(sol) == Vector{Float64}
+  ### below is TBD
+  # @test model.meta.nvar == 5
+  # @test model.nls_meta.nequ == 202
+  # @test all(model.meta.x0 .== 1)
+  # @test length(findall(x -> x .!= 0, sol)) == 2
```

Comment:
What is there to determine here?

Reply:
I was considering putting the derivative tests in here, where we used ReverseADNLP (I believe). Thoughts on keeping in?

Reply:
ReverseADNLPModels isn't anywhere near stable. We shouldn't rely on it other than to check things locally ourselves.

Reply:
Sounds good I can get these removed. Theres still the deriv_check function floating around somewhere, but I can find and remove it.